specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json

```
@@ -45,7 +45,10 @@
   "paths": {
     "/availability": {
       "get": {
-        "operationId": "GetAvailability",
+        "tags": [
+          "TimeSeries"
+        ],
+        "operationId": "TimeSeries_AvailabilityGet",
```

```suggestion
        "operationId": "TimeSeries_GetAvailability",
```
In both English generated docs and generated code, "Get Something" and GetSomething() is more common, consistent and more readable (especially that in other places we do put verb first).